projects/igniteui-angular/schematics/ng-add/index.ts

```diff
@@ -49,6 +52,25 @@ function LogIncludingDependency(context: SchematicContext, pkg: string, version:
   context.logger.log('info', `Including ${pkg} - Version: ${version}`);
 }
 
+function promptAddPolyfills(tree: Tree, options: Options) {
+  if (options['polyfills']) {
+    const targetFile = 'src/polyfills.ts';
+    if (!tree.exists(targetFile)) {
+      throw new Error(`${targetFile} not found in the src folder.`);
+    }
+
+    // Match all commented import statements that are core-js/es6/*; core-js/es7/*; web-animations-js
+    const pattern = /\/{2}\s{0,}\w+\s{0,}\'(\w+\-\w+\/(\w+)(6|7)\/.+|\w+\-\w+\-\w+\';)/g;
```

Comment:
That's way too broad. You _can_ limit it by using core-js in the regex and a super simple text replace for the web animations plugin.

Keep in mind there's a `'core-js/es6/array'` import that's not related and there might be more. For this specific usage even a pattern list of the specific imports doesn't seem like a bad option.